content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,381 @@
+
+- directly submitted as a _fault_ proof by a _reporting_ validator
+- promoted from accusations where they are:
+  - reported as an _accusation_ proof, submitted by a _reporting_ validator against an _offending_ validator
+  - defended by an _innocence_ proof, submitted by the _offending_ validator within a proof submission window measured in blocks
+  - promoted to _fault_ by protocol on individual block finalisation if not refuted by _innocence_ and the _offending_ validator does not already have a _fault_ with a higher or equivalent severity. 
+
+Slashing penalties are computed by protocol and  applied for proven faults at epoch end. The penalty amount is computed based on a base slashing rate and slashing factors including the total number of slashable offences committed in the epoch and the individual _offending_ validator's own slashing history.
+
+Slashing is applied as part of the state finalisation function:
+
+- As each block is finalised, AFD will: promote within protocol constraints new _accusations_ to proven _faults_ after expiry of an _innocence_ proof submission window
+- As the last block of an epoch is finalised, AFD will: apply accountability for _faults_ to _offending_ validators, slashing [self-bonded](/glossary/#self-bonded) and [delegated](/glossary/#delegated) stake  according to Autonity's [Penalty-Absorbing Stake (PAS)](/glossary/#penalty-absorbing-stake-pas) model.
+
+Rewards are paid for reporting a slashable _fault_ at epoch end. The _offending_ validator's share of the epoch's staking rewards is forefeited and paid to the  _reporting_ validator `treasury` account for distribution along with the staking rewards at epoch end.
+
+### Accountability prerequisites 
+
+To participate in AFD a [validator](/glossary/#validator) must be a [consensus committee](/glossary/#consensus-committee) member.
+
+## Accountability and Fault Detection protocol
+
+AFD roles, core concepts, and the lifecycle of accountability event processing from accusation to slashing.
+
+### Validator roles
+
+As consensus committee members, validators play the following roles in AFD processing:
+
+- _reporting_: as the validator reporting a suspected rule infraction and submitting new _accusation_ proofs on-chain
+- _offending_: as the validator committing a suspected rule infraction and submitting new _innocence_ proofs on-chain
+- _committee member_: as a validator in the consensus committee executing Autonity's consensus protocol and for AFD handling and processing accountability events, maintaining system state for accountability events, and computing and applying slashing penalties.
+
+Validators play the following economic roles in AFD processing:
+
+- _loss_: loss of stake, validator reputation, staking reward revenue as the _offending_ validator of a slashed _fault_
+- _beneficiary_: gain of slashing reward as the _reporting_ validator of a slashed _fault_.
+
+Autonity community play the following economic role in AFD processing:
```

Comment:
Do we mean the delegators here?

Reply:
No. The reporting validator gets the slashing reward itself, it's not distributed by the reporting validator to its stake delegators. Youssef clarified that to me.

I added the mention of community here to clarify that any slashed stake token goes to community funds.

Reply:
Do you think further editing or clarification is required in the docs here @lorenzo-dev1 ?

Reply:
I would just clarify that the reporting validators gets the /rewards/ of the slashed validators, but not the slashed amount (which goes to community).

More specifically this sentence under the validator role:
`beneficiary: gain of slashing reward as the reporting validator of a slashed fault.`

can be misinterpreted as the reporter getting the slashed amount.